ropetest/refactor/importutilstest.py

```diff
@@ -1900,6 +1900,20 @@ def test_organizing_imports_all_star_no_infinite_loop(self):
         pymod = self.project.get_pymodule(self.mod)
         self.assertEqual(expected, self.import_tools.organize_imports(pymod))
 
+    def test_organizing_indirect_all_star_import(self):
+        self.mod1.write("some_name = 1")
+        self.mod2.write("__all__ = ['some_name']")
+
+        code = expected = dedent("""\
+            from mod1 import *
+
+            from mod2 import __all__
+        """)
+        self.mod3.write(code)
+
+        pymod = self.project.get_pymodule(self.mod3)
+        self.assertEqual(expected, self.import_tools.organize_imports(pymod))
+
```

Comment:
@thomkeh I added this case, which seems to simulate a possible scenario on how you might end up with the error you are seeing. Can you check if the issue in your codebase is caused by something similar?

Reply:
Hm, I'm definitely importing some constants, like your
```
some_name = 1
```
and then I add it to `__all__` (and import it with star import somewhere else), but I don't think I have something like `from mod2 import __all__`. And I don't think I'm putting anything in `__all__` that wasn't defined in that file.

Reply:
In case that was a bit confusing. Here an example:

mod1.py
```
__all__ = ["PI"]
PI = 3.14
```

mod2.py
```
from mod1.py import *
```